### PR TITLE
[PT FE]: support frobenius norm and fix aten::norm

### PR DESCRIPTION
**Pull request by eaidova**
### Details:
 - *support aten::frobenius_norm*

### Tickets:
 - *101102*
tested on clip model

### CODE REVIEW

File: tests/layer_tests/pytorch_tests/test_norm.py [hunk 1]
```diff
@@ -32,12 +29,87 @@ def forward(self, input_data):
...
+    def test_norm_tensor(self, ie_device, precision, ir_version, p, dim, keepdim):
+        self._test(*self.create_model_tensor_norm(p, dim, keepdim),
+                   ie_device, precision, ir_version)
+
+
+class TestFrobeniusNorm(PytorchLayerTest):
+    def _prepare_input(self, out=False):
+        if not out:
+            return (np.random.randn(1, 2, 3).astype(np.float32),)
+        x = np.random.randn(1, 2, 3).astype(np.float32)
```
Comment (mvafin) [inline tests/layer_tests/pytorch_tests/test_norm.py:79]: Tensor is too small to verify the correctness of norm calculation, could you increase it to at least [10,12,14].

File: tests/layer_tests/pytorch_tests/test_norm.py [hunk 2]
```diff
@@ -32,12 +29,87 @@ def forward(self, input_data):
...
+        ref_net = None
+
+        return aten_frobenius_norm(dim, keepdim, out), ref_net, "aten::frobenius_norm"
+
+    @pytest.mark.nightly
+    @pytest.mark.precommit
+    @pytest.mark.parametrize('dim', [(1, ), (0, ), (-1, ), (0, 1), (1, 0)])
+    @pytest.mark.parametrize('keepdim', [True, False])
+    @pytest.mark.parametrize("out", [False, True])
+    def test_frobenius_norm(self, ie_device, precision, ir_version, dim, keepdim, out):
```
Comment (mvafin) [inline tests/layer_tests/pytorch_tests/test_norm.py:108]: What if the input to this operation is integer type? Does pytorch support such case?

**Review by mvafin — CHANGES_REQUESTED** [review]
In general looks good, please fix code style and small comments

File: tests/layer_tests/pytorch_tests/test_norm.py (hunk 2 above)
Comment (eaidova) [reply tests/layer_tests/pytorch_tests/test_norm.py:108]: PyTorch does not support such case:
RuntimeError: forbenius_norm(): input dtype should be either floating point or complex. Got Int instead

File: tests/layer_tests/pytorch_tests/test_norm.py (hunk 1 above)
Comment (eaidova) [reply tests/layer_tests/pytorch_tests/test_norm.py]: updated

**Review by mvafin — APPROVED** [review]

**Pull request merged**